### ISSUE TITLE
Zoomed image isn't displayed the second time

### ISSUE THREAD

**Issue by machour** [gitpoint/git-point#343]
Found this while testing the app on iPhone X simulator, it seems that ios fails the second time you tap a ZoomedImage (from a PR or a profile screen). 
I was also able reproduced it on iPhone 7 and 8 simulators.

Can someone with a real device confirm this?

Error logged the second time:
```
2017-09-19 20:48:04.617844+0100 GitPoint[24593:16853987] [Unknown process name] CGAffineTransformInvert: singular matrix.
```

**Comment by machour** [comment]
Found this freshly opened PR that fixes the bug in react-native-photo-view repo : https://github.com/alwx/react-native-photo-view/pull/103

**Comment by lex111** [comment]
I did this component, then I did not have the opportunity to check on iOS, although it worked, but probably no one checked if the image opens the second time, but it's good that there is a PR that fixes it, only the author of this library does not actively take PR's, so we'll have to wait, or we can make a fork of this library..

**Comment by machour** [comment]
Merged \o/ https://github.com/alwx/react-native-photo-view/commit/3d36136ca2cd4c39270541ffa7e951a8eed92008

Fix should be present in 1.5.1

**Comment by lex111** [comment]
Oh, this is very cool, then we just need to update the version of this dependency.
